_episodes/09-practical-examples.md

```diff
@@ -7,5 +7,86 @@ questions:
 objectives:
   - Use docker in combination with real research software
 keypoints:
--  
+  - Docker can be a solution to include the legacy script into your workflow with minimum effort.
+  - You can use intermediate files to let different Docker containers talk to each other.
 ---
+
+You are a social science researcher and are trying to establish how many people in a list are below the allowed drinking age.
```

Comment:
Could you add `>` and `> >` before the lines of the exercise so it is formatted properly?
Also, it needs to end with `{: .challenge}

Reply:
Could you add a title to the exercise?